### ISSUE TITLE
Personal details coutry not Automatically used

### ISSUE THREAD

**Issue by SiFuMax09** [WemXPro/tracker#269]
### Product

WemX

### Describe the issue

A bug With the Personal details that the country option frm the user settings will not be Automatically set to the users options

### Product Version

1.8.1

### To Reproduce

First set a Country that is not ``Afghanistan`` try to buy something and it will set the country to Afghanistan

### Expected Behaviour

It will Automatically set the country that is set in the user options

### Screenshots

![image](https://github.com/WemXPro/tracker/assets/131544883/0a624c84-1a4d-454f-b137-a145f04d1de3)
![image](https://github.com/WemXPro/tracker/assets/131544883/5cbe8ce1-1a45-4319-a9d5-b9e15423cd28)

**Comment by VanillaChan6571** [comment]
Known Bug since launch of wemx 1.0.0, unknown fix except for rewrite.

**Comment by Mubeen142** [comment]
Hi thank you for reporting this issue.

It seems there was a small error with how the countries were selected. I have updated the code, it works as following now:

1. If an application is using CloudFlare, we can retrieve the users country from CloudFlare and set that as the default
2. If an application is not using CloudFlare, it will set the country to the users country in their account settings

In addition, I have also added a method for applications using CloudFlare, that when a user signs up the application automatically sets their country to streamline to process

This will be available next update.
